### ISSUE TITLE
how to select text in TextBox by mouse left button

### ISSUE THREAD

**Issue by Charltsing** [aelyo-softworks/Wice#7]
i set IsEditable = true , it can select text by keyboard, but can not select by mouse left button

how to select text in TextBox by mouse left button

[can not select by mouse.zip](https://github.com/aelyo-softworks/Wice/files/13226663/can.not.select.by.mouse.zip)

**Comment by smourier** [comment]
It's not currently supported by RichTextBox, but should work fine with TextBox by mouse and keyboard:

![image](https://github.com/aelyo-softworks/Wice/assets/5328574/f00e2f63-39d2-4c02-83a5-02ef8088bab9)

Maybe you have no SelectionBrush or by default it's transparent (but in this case it shouldn't work by keyboard...)

**Comment by Charltsing** [comment]
it is my demo project (vs2022), with lastest wice source

do you try it ?

[can not select by mouse (2).zip](https://github.com/aelyo-softworks/Wice/files/13227061/can.not.select.by.mouse.2.zip)
[WiceDemo.zip](https://github.com/aelyo-softworks/Wice/files/13227078/WiceDemo.zip)

**Comment by smourier** [comment]
The issue was caused by your code using `WindowsFunctions.EnableMouseInPointer()` which calls this Windows API https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-enablemouseinpointer that asks Windows to send `WM_POINTER `messages instead of `WM_MOUSE `messages.

Using the pointer instead of mouse wasn't fully supported. I have enhanced Wice (https://github.com/aelyo-softworks/Wice/commit/1ed33cb14b75bc077e518729473cae33f76504ee) and it should work better now.

The gallery also now allows to set it manually if unset at startup (it cannot be unset once set):

![image](https://github.com/aelyo-softworks/Wice/assets/5328574/4b189c43-f48e-420d-91da-5961268ff478)
